src/vs/workbench/contrib/tasks/browser/abstractTaskService.ts

```diff
@@ -294,7 +294,9 @@ export abstract class AbstractTaskService extends Disposable implements ITaskSer
 		}));
 		this._taskRunningState = TASK_RUNNING_STATE.bindTo(_contextKeyService);
 		this._onDidStateChange = this._register(new Emitter());
-		this._registerCommands().then(() => TaskCommandsRegistered.bindTo(this._contextKeyService).set(true));
+		if (!isWeb) {
```

Comment:
`isWeb` isn't imported, but this also includes remote cases connected via a browser (ie. Codespaces). Do you mean to check `IRemoteAgentService.getConnection()?.remoteAuthority`?

Reply:
I think I should check both `isWeb` and `!remoteAuthority` because this should work in remotes like WSL